src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/CCSvcBeanMapperTest.CaseContainerDTO.json

```diff
@@ -45,5 +45,5 @@
     }
   ],
   "collectionExerciseId": "22684ede-7d5f-4f53-9069-2398055c61b2",
-  "surveyType": "x"
+  "surveyType": "CENSUS"
 }
```

Comment:
just while you are here, please add a new line at the end to get of the github warning about no end-of-line, thanks.

Reply:
Added new line at the end